src/widgets/search-box/search-box.js

```diff
@@ -168,6 +185,10 @@ function searchBox({
         return actualEvent => actualEvent.keyCode === expectedKeyCode && fnToExecute(actualEvent);
       }
 
+      function getQueryAndCall(fnToExecute) {
```

Comment:
I renamed to "func" and moved them at the bottom of the file for now to indicate that there's no shared state